### PR TITLE
Gradient Circle

### PR DESCRIPTION
**Pull request by sinanku**
gradient

### CODE REVIEW

**Comment by sinanku** [comment]
@skywinder It looks like it's complaining about these two lines

`super.init(coder: aDecoder)` and `gaugeLayer = getGauge(rotateAngle: rotate / 10 * M_PI)`

which I changed to 

`super.init(coder: aDecoder)!` and `gaugeLayer = getGauge(rotate / 10 * M_PI)`

Current project won't compile without the changes I made to run locally, thoughts?

**Comment by skywinder** [comment]
@sinanulkuatam your example looks pretty good! But what the reason to force eusers run pod install? why not just add it as internal framework?
About errors:
please wait for merge of #32,  I hope it will not make merge conflicts. It should remove some problems and will update syntax a bit

**Comment by sinanku** [comment]
@skywinder yes you're right about adding it as an internal framework, I'll make that change today

**Comment by sinanku** [comment]
@skywinder a couple changes, I removed the Pods and all dependencies around the build for it.  I had a couple issues embedding it directly as it pointed to my own derived data, so instead of running Example.xcodeproj you can run GaugeKit.xcworkspace as it was originally suggested on the readme and change your scheme to run the example.  I will update the docs to take this into account.

**Comment by sinanku** [comment]
@skywinder let me know if there are any issues with the branch/PR

File: Example/Example/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -423,7 +438,7 @@
                                             </userDefinedRuntimeAttribute>
                                         </userDefinedRuntimeAttributes>
                                     </view>
-                                    <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="q6f-rg-MaX" userLabel="GaugeMid" customClass="Gauge" customModule="GaugeKit">
+                                    <view contentMode="scaleToFill" misplaced="YES" translatesAutoresizingMaskIntoConstraints="NO" id="q6f-rg-MaX" userLabel="GaugeMid" customClass="Gauge" customModule="GaugeKit">
```
Comment (skywinder) [inline Example/Example/Base.lproj/Main.storyboard:421]: Can you kindly mix misplaced views?

**Comment by skywinder** [comment]
There is also broken test in this build: https://travis-ci.org/skywinder/GaugeKit/builds/131408468#L310

"   Missing argument label 'rotateAngle:' in call"

File: GaugeKit/Gauge.swift [hunk 2]
```diff
@@ -87,6 +87,18 @@ public class Gauge: UIView {
             updateLayerProperties()
         }
     }
+    @IBInspectable public var topGradientColor: UIColor = UIColor.greenColor() {
+        didSet {
+            resetLayers()
+            updateLayerProperties()
+        }
+    }
+    @IBInspectable public var bottomGradientColor: UIColor = UIColor.redColor() {
```
Comment (skywinder) [inline GaugeKit/Gauge.swift]: Are you sure, that we have to add 2 additional properties? Why you don't use `startColor` and `endColor` properties in `Gauge.swift`? they are useless in circle yet. I think too many properties can confuse users.

File: Example/Example/Base.lproj/Main.storyboard (hunk 1 above)
Comment (sinanku) [inline Example/Example/Base.lproj/Main.storyboard:421]: Not sure why they are now misplaced, didn't change the storyboard and the demo project runs across all device sizes.  Will check what the issue is however

Edit: Updated current Main.storyboard to Master

File: GaugeKit/Gauge.swift (hunk 2 above)
Comment (sinanku) [inline GaugeKit/Gauge.swift]: I agree, removed

File: .gitignore [hunk 3]
```diff
@@ -30,4 +31,3 @@ Carthage
 # Note: if you ignore the Pods directory, make sure to uncomment
 # `pod install` in .travis.yml
 #
-# Pods/
```
Comment (skywinder) [inline .gitignore:33]: Please, don't add it to gitignore

File: README.md [hunk 4]
```diff
@@ -17,7 +17,7 @@
 
 ## Example Usage
 
-Just open `GaugeKit.xcworkspace` and run Example project.
+Open `GaugeKit.xcworkspace` and change the scheme to 'Example' and run Example project.
```
Comment (skywinder) [inline README.md:20]: That's good addition

**Review by skywinder — CHANGES_REQUESTED** [review]
Hi. Sorry for late reply. As I can see you make a good addition to the project. But sadly it's updated to swift 3.0. I would like to review your PR if you  can update it to current status of the project/

**Review by skywinder — APPROVED** [review]

**Comment by skywinder** [comment]
Thank you for PR. 
And sorry for late reply.
The project is moved to Swift 3. I would like to review your changes if you will able to synchronize it with current version.

**Comment by sinanku** [comment]
I will have time later this week to work on it, thanks for the update!

**Comment by skywinder** [comment]
I'd be more than happy to merge a PR adding this feature, it seems like it would be useful for sure 👍
Please let me know, you going to update it to up-to date version and fix minor things!

**Comment by sinanku** [comment]
@skywinder I'll be able to take a look in a few weeks, wrapping up grad school at the moment!

**Comment by skywinder** [comment]
CLosed in favor of #68

**Pull request closed without merging**